specification/definitions/Shipment.json

```diff
@@ -152,6 +152,86 @@
             }
           }
         },
+        "customs": {
+          "type": "object",
+          "properties": {
+            "content_type": {
+              "type": "string",
+              "enum": [
+                "merchandise",
+                "sample_merchandise",
+                "returned_merchandise",
+                "documents",
+                "gifts"
+              ],
+              "example": "merchandise"
+            },
+            "invoice_number": {
+              "type": "string",
+              "example": "9000"
+            },
+            "items": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "required": [
+                  "sku",
+                  "description",
+                  "item_value",
+                  "quantity",
+                  "hs_code",
+                  "origin_country_code"
+                ],
+                "additionalProperties": false,
+                "properties": {
+                  "sku": {
+                    "type": "string",
+                    "minLength": 1,
+                    "example": "123456789"
+                  },
+                  "description": {
+                    "type": "string",
+                    "minLength": 1,
+                    "example": "OnePlus X"
+                  },
+                  "item_value": {
+                    "$ref": "../includes/price.json"
+                  },
+                  "quantity": {
+                    "type": "number",
+                    "example": 2
+                  },
+                  "hs_code": {
+                    "type": "string",
+                    "minLength": 1,
```

Comment:
HS codes are composed of numbers and periods (for as far as I can see), so we could add a pattern here:
`^[0-9\.]+$`